<?xml version="1.0"?>
<!-- This launch file can be used to launch an gazebo instance. -->
<!-- Simply call this launch file with the default parameters. -->
<launch>
  <rosparam command="load" file="$(find infinite_interaction)/config/admittance_params.yaml"/>
  <node name="admittance_control" pkg="infinite_interaction" type="admittance_controller" output="screen"/>
</launch>
